src/locales/en/translation.json

```diff
@@ -94,5 +94,10 @@
       "Website": "Website",
       "Menu": "Menu"
     }
+  },
+  "donationBox": {
```

Comment:
We already have a Donation Pool section here, though the text is slightly different. Maybe we can at least use the same Chinese?